RELEASE_NOTES.md

```diff
@@ -1,5 +1,46 @@
 # Release Notes
 
+## stable-1.15-2.1.0, stable-1.16-2.1.0, stable-1.17-2.1.0
+
+* Kibana: Importing an Audit-Search dashboard and 2 Audit-Search visualizations to help search through audit logs. ([#254](https://github.com/mesosphere/kubernetes-base-addons/pull/254), [@alejandroEsc](https://github.com/alejandroEsc))
+* [traefik] fix the velero-minio entrypoint to inherit global ssl and proxy protocol configurations ([#259](https://github.com/mesosphere/kubernetes-base-addons/pull/259), [@jieyu](https://github.com/jieyu))
+* opsportal bump chart to 0.3.13 ([#261](https://github.com/mesosphere/kubernetes-base-addons/pull/261), [@juliangieseke](https://github.com/juliangieseke))
+* azuredisk-csi-driver
+
+  Kubernetes upgrade to 1.17.x
+
+  The change from v1alpha1 to v1beta1 snapshot APIs is not backward compatible.
+  If you already have a Konvoy Cluster with Kubernetes version < 1.17 you would need to follow these instructions before upgrading the Konvoy Cluster to kubernetes 1.17:
+
+  **Note: The underlying snapshots on the storage system will be deleted in the upgrade process!!!**
+
+  1. Delete volume snapshots created using v1alpha1 snapshot CRDs and external-snapshotter sidecar controller.
+
+  1. Set the replicas to 0 for the statefulSet `ebs-csi-controller`
+
+     ```shell
+     kubectl scale statefulset -n kube-system ebs-csi-controller --replicas=0`
+     ```
+
+  1. Uninstall v1alpha1 snapshot CRDs
+
+     ```shell
+     kubectl delete customresourcedefinitions.apiextensions.k8s.io volumesnapshotclasses.snapshot.storage.k8s.io
+     kubectl delete customresourcedefinitions.apiextensions.k8s.io volumesnapshotcontents.snapshot.storage.k8s.io
+     kubectl delete customresourcedefinitions.apiextensions.k8s.io volumesnapshots.snapshot.storage.k8s.io
+     ```
+
+  1. Rollout upgrade to Kubernetes 1.17
+  1. Set the replicas back to what it was before (e.g. 2) for the statefulSet `ebs-csi-controller`
+
+     ```shell
+     kubectl scale statefulset -n kube-system ebs-csi-controller --replicas=2
+     ```
+
+   ([#262](https://github.com/mesosphere/kubernetes-base-addons/pull/262), [@sebbrandt87](https://github.com/sebbrandt87))
```

Comment:
@sebbrandt87 What does this mean from the standpoint of someone bumping their kubeaddons from 2.0.0 to 2.1.0 in their cluster.yaml. This is going to cause data loss?